src/pages/dashboard/index.tsx

```diff
@@ -242,14 +242,19 @@ const Home: NextPage = () => {
                   </StatHelpText>
                 </StatNumber>
               ) : (
-                <Skeleton height="30px" mt="5" borderRadius="full" />
+                <Skeleton
+                  height={{ xl: '30px', base: '18px' }}
+                  mt={{ md: '5', base: '0' }}
```

Comment:
This is better written this way.
```suggestion
                  mt={{md: '5' }}
```